ui/ui-components/components/appSelectorList.tsx

```diff
@@ -1,6 +1,34 @@
 import Selector, { BadgeProp } from "./selector";
 import { CardDeck } from "react-bootstrap";
 import { useRouter } from "next/router";
+import { Actions } from "../utils/apiData";
+
+type SelectableApp = Actions.AppOptions["types"][number];
+type SelectablePackage = Actions.PluginsAvailableList["plugins"][number];
+type SelectableSource =
+  Actions.SourceConnectionApps["connectionApps"][number]["app"];
+type SelectableDestination =
+  Actions.DestinationConnectionApps["connectionApps"][number]["app"];
```

Comment:
This component was a bit wild before - handling 3 types of API responses without any typing.... now we have types!